src/confcom/azext_confcom/docs/v1_json_file_format.md

```diff
@@ -0,0 +1,221 @@
+# v1 `--input`, `-i` json file format
+
+This document provides a comprehensive reference for the JSON configuration format used by the `confcom` extension's `--input`, `-i` flag.
+This schema allows you to define container properties and fragment specifications to be used with `acipolicygen` and `acifragmentgen` including usage with `VN2`.
+
+## Schema Overview
+
+The configuration file uses a structured JSON format with the following main sections:
+
+| Field        | Type    | Required                                      | Description                                                                                   |
+|--------------|---------|-----------------------------------------------|-----------------------------------------------------------------------------------------------|
+| `version`    | string  | Yes                                           | Policy framework version identifier (currently "1.0")                                                   |
+| `scenario`   | string  | No                                            | Adds appropriate mounts and environment variables for different deployment scenarios. If deploying to `vn2`, this field is required |
+| `fragments`  | array   | Yes (either `containers` or `fragments` must be defined) | Specifies policy fragments to include                                                          |
+| `containers` | array   | Yes (either `containers` or `fragments` must be defined) | Container configurations to deploy                                                             |
+
+## Detailed Field Reference
+
+### Top-Level Fields
+
+#### `version`
+
+- **Type**: String
+- **Required**: Yes
+- **Description**: Version identifier for the output policy format
+- **Allowed Values**: "1.0"
+- **Example**: `"version": "1.0"`
+
+#### `scenario`
+
+- **Type**: String
+- **Required**: No
+- **Description**: Adds appropriate mounts and environment variables for different deployment scenarios.
+If deploying to `vn2`, this field is required.
+The default value is `aci`
+- **Allowed Values**: "vn2", "aci"
+- **Example**: `"scenario": "vn2"`
+
+#### `fragments`
+
+- **Type**: Array of objects
+- **Required**: Yes (either `containers` or `fragments` must be defined)
+- **Description**: Policy fragments that should be included and imported when enforcing the security policy
+
+#### `containers`
+
+- **Type**: Array of objects
+- **Required**: Yes (either `containers` or `fragments` must be defined)
+- **Description**: Container specifications for deployment
+
+### Fragment Objects
+
+These objects define all the information necessary for importing and using a fragment.
+
+#### fragments
```

Comment:
Put this in in the fragments section on line 39, then you won't need to repeat lines 57-59